plinth/action_utils.py

```diff
@@ -121,13 +122,16 @@ def service_reload(service_name):
 def webserver_is_enabled(name, kind='config'):
     """Return whether a config/module/site is enabled in Apache."""
     option_map = {'config': '-c', 'site': '-s', 'module': '-m'}
-    try:
-        # Don't print anything on the terminal
-        subprocess.check_output(['a2query', option_map[kind], name],
-                                stderr=subprocess.STDOUT)
-        return True
-    except subprocess.CalledProcessError:
-        return False
+    if spawn.find_executable('a2query'):
+        try:
+            # Don't print anything on the terminal
+            subprocess.check_output(['a2query', option_map[kind], name],
+                                    stderr=subprocess.STDOUT)
+            return True
+        except subprocess.CalledProcessError:
+            return False
+    else:
+        print("web configuration is not enabled,please reinstall apache2 web server")
```

Comment:
I don't think we want to print anything here. This function should only return True or False (or possibly, raise an exception). I would suggest just returning False--we can assume that the web conf is disabled if there's no apache2 installed.